local/include/etc/confluent/docker/launch

```diff
@@ -44,6 +44,17 @@ echo "===> Launching kafka-rest ... "
 kafka-rest-start /etc/kafka-rest/kafka-rest.properties & # your second application
 P2=$! # capture PID of the process
 
+# Function to handle cleanup
+cleanup() {
+  echo "Terminating processes..."
+  kill $P1 $P2
+  wait $P1 $P2 2>/dev/null
+  exit 1
+}
+
+# Trap SIGINT (Ctrl+C) and call cleanup
+trap cleanup SIGINT
```

Comment:
`STOPSIGNAL SIGINT` is required in Dockerfile to make this code working as expected. Refer to https://github.com/confluentinc/kafka-images/pull/401